lib/mail/multibyte/unicode.rb

```diff
@@ -390,11 +402,3 @@ def database
     end
   end
 end
-
-unless defined?(ActiveSupport)
-  module ActiveSupport
-    unless const_defined?(:Multibyte)
-      Multibyte = Mail::Multibyte
-    end
-  end
-end
```

Comment:
These lines of code seems to have been added here: b9fd65b3561b0c1b3a2fca502bbfe1908e74591b. The intent of that commit was "unicode dat file is a marshal of some ActiveSupport". This pull request removes that dependency by building the dat file on our own, without ties to ActiveSupport.

I'm no expert in neither this mail gem nor encoding, but to me this seems like the solution which should have been done three years ago.